shared/people/todo/container.tsx

```diff
@@ -15,40 +15,74 @@ import openURL from '../../util/open-url'
 type TodoOwnProps = {
   badged: boolean
   confirmLabel: string
-  dismissable: boolean
   icon: IconType
   instructions: string
+  subText: string
   todoType: Types.TodoType
+  buttons: Array<TaskButton>
 }
 
 const installLinkURL = 'https://keybase.io/download'
 const onSkipTodo = (type: Types.TodoType, dispatch) => () => dispatch(PeopleGen.createSkipTodo({type}))
 const mapStateToProps = state => ({myUsername: state.config.username || ''})
 
+type TodoButtonOpts = {
```

Comment:
It'd be nice for `makeDefaultButtons` to be less dependent on the structure of `ownProps` and `dispatchProps` below. could these params be separated out?